app/controllers/vacation_requests_controller.rb

```diff
@@ -0,0 +1,44 @@
+class VacationRequestsController < ApplicationController
+  respond_to :json
+
+  before_action :authenticate_user!
+
+  # GET
+  # /resource
+  # All the user's vacation requests
+  def index
+    render json: current_user.vacation_requests
+  end
+
+  # POST
+  # /resource
+  def create
+    @vacation_request = VacationRequest.new vacation_requests_params
+    # Convert JS Date/Time in miliseconds to Ruby Date format
+    @vacation_request.start = Time.at(@vacation_request.start / 1000)
+    @vacation_request.user_id = current_user.id
+    if @vacation_request.save
+      respond_with @vacation_request
+    else
```

Comment:
I think there should be:
`flash[:success] = 'Vacation request has been successfully created' if @vacation_request.save`
`respond_with @vacation_request`, according to responders gem.
Also remove comments from generated rails controllers.